seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/utils/SystemUtil.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.seatunnel.core.starter.utils;
+
+import org.apache.commons.lang3.SystemUtils;
+
+public class SystemUtil {
+      
+    public static String GetOsType() {
+        String os_type="";
+        
+        if (SystemUtils.IS_OS_WINDOWS) {
+           os_type="Windows";
+        } else if (SystemUtils.IS_OS_MAC) {
+           os_type="Mac";
+        } else if (SystemUtils.IS_OS_LINUX) {
```

Comment:
switch?

Reply:
Do you mean change the code  to  the code style  "switch ... case" ?  If this is true, I'll modify as you suggested.

Reply:
I don't think  "switch... case" is needed here, because there are many different properties from SystemUtils(e.g. IS_OS_LINUX, IS_OS_WINDOWS,,,etc),  if  SystemUtils can return unique property "OS_TYPE" then we can change to Switch... case style easily.